pkg/apis/openebs.io/scp/v1alpha1/affinity.go

```diff
@@ -0,0 +1,42 @@
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Affinity is a group of affinity scheduling rules
+type Affinity struct {
+
+	// Describes volume affinity scheduling rules
```

Comment:
VolumeAffinity helps to place a volume in the same topology(storage cohort) where old volumes are present, if old volume not found the it can go to any cohort. Old volumes are identified using `LabelSelector`. Using this we can ensure that if any pod is requesting for 2 volumes then 2 volumes will be scheduled in the same storage cohort topology. ie - if `mongo-0` replica of sts `mongo` is requesting for 2 volumes `data-0` and `log-0`, both the volumes will be scheduled in the same storage cohort topology.

Reply:
Please add this in the comment.

Reply:
also add these are ANDed.